frmts/vrt/vrtderivedrasterband.cpp

```diff
@@ -743,6 +747,73 @@ bool VRTDerivedRasterBand::InitializePython()
     return true;
 }
 
+CPLErr
+VRTDerivedRasterBand::GetPixelFunctionArguments(
+  const CPLString& osMetadata,
+  std::vector<std::pair<CPLString, CPLString>>& oAdditionalArgs)
+{
+
+    CPLXMLNode* oArgs = CPLParseXMLString(osMetadata);
```

Comment:
to avoid memory leak:
```suggestion
    auto poArgs = CPLXMLTreeCloser(CPLParseXMLString(osMetadata));
```